network_tools/test/network_tools_test.dart

```diff
@@ -1,3 +1,5 @@
-const port = 22; //ssh (22) is for github actions and dns (53) for local machine
-const timeout = Duration(milliseconds: 500);
+const port = 53; //ssh (22) is for github actions and dns (53) for local machine
+const timeout = Duration(milliseconds: 3000);
+const lastHostId =
+    1024; //keep it 128 for local tests, and 1024 for github actions
```

Comment:
Users will not know that they need to change it in order for the test to pass.
If you can find this number problematically it will be best.
Otherwise add it into the documentation of tests.